conf/nginx.conf

```diff
@@ -4,11 +4,6 @@ location __PATH__/ {
   # Path to source
   alias __FINALPATH__/;
 
-  # Force usage of https
-  if ($scheme = http) {
-    rewrite ^ https://$server_name$request_uri? permanent;
-  }
-
```

Comment:
Why remove this section ? Is it now done at a whole-domain level ?

Reply:
NGINX HTTPS now part of the core

Reply:
OK !